### DISCUSSION TITLE
Build Controllers from Source and use the same in bootstrap

### DISCUSSION THREAD

**Discussion by lakshman-au** [fluxcd/flux2#4204] [General]
I am inside a corporate network, my AKS is having  access to Gitlab repo(again inside network). when i  tried to bootstrap, it is try to use the controllers which is not available (as it cannot pull ). I want to push controllers image to ACR and use it.  I want to know how to configure bootstrap to use images in my ACR.

**Comment by stefanprodan** [comment]
Check out the air gap installation guide https://fluxcd.io/flux/installation/configuration/air-gapped/
